### PR TITLE
Add changelog for release 2.0

### PR DESCRIPTION
**Pull request by xing-yang**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
Add changelog for release 2.0.
Also removes changelog for 1.3 as we don't have a 1.3 release.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

### CODE REVIEW

File: CHANGELOG-2.0.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+## New Features
+
+- Add prometheus metrics to CSI external-snapshotter under the /metrics endpoint. This can be enabled via the "--metrics-address" and "--metrics-path" options. ([#227](https://github.com/kubernetes-csi/external-snapshotter/pull/227), [@saad-ali](https://github.com/saad-ali))
+- Split the external-snapshotter sidecar controller into two controllers, a snapshot-controller and an external-snapshotter sidecar. Only the external-snapshotter sidecar should be deployed with the CSI driver. ([#182](https://github.com/kubernetes-csi/external-snapshotter/pull/182), [@xing-yang](https://github.com/xing-yang))
+- Add a finalizer on VolumeSnapshot object to protect it from being deleted when it is bound to
+ a VolumeSnapshotContent. ([#182](https://github.com/kubernetes-csi/external-snapshotter/pull/182), [@xing-yang](https://github.com/xing-yang))
+
+### API Changes
+
+- Changes VolumeSnapshot CRD version from v1alpha1 to v1beta1. ([#139](https://github.com/kubernetes-csi/external-snapshotter/pull/139), [@xing-yang](https://github.com/xing-yang))
```
Comment (msau42) [inline CHANGELOG-2.0.md]: Should this also go under breaking changes since alpha is no longer supported

File: CHANGELOG-1.3.md [hunk 2]
```diff
@@ -1,9 +0,0 @@
-# Changelog since v1.2.0
-
-## Breaking Changes
-
-- Add a prefix "external-snapshotter-leader" and the driver name to the snapshotter leader election lock name. Rolling update will not work if leader election is enabled because the lock name is changed in v1.3.0. ([#129](https://github.com/kubernetes-csi/external-snapshotter/pull/129), [@zhucan](https://github.com/zhucan))
```
Comment (msau42) [inline CHANGELOG-1.3.md]: Are these changes still new for 2.0?

Comment (xing-yang) [reply CHANGELOG-1.3.md]: Good point.  This one is not backported.

File: CHANGELOG-2.0.md (hunk 1 above)
Comment (xing-yang) [reply CHANGELOG-2.0.md]: Sure

**Comment by xing-yang** [comment]
@msau42 addressed your comments.  PTAL.

File: CHANGELOG-1.3.md [hunk 3]
```diff
@@ -1,9 +0,0 @@
...
-
-## Other Notable Changes
-
-- Snapshotter will no longer call ListSnapshots if the CSI Driver does not support this operation. ([#138](https://github.com/kubernetes-csi/external-snapshotter/pull/138), [@ggriffiths](https://github.com/ggriffiths))
```
Comment (msau42) [inline CHANGELOG-1.3.md]: Also this one?

Comment (xing-yang) [reply CHANGELOG-1.3.md]: This one is already backported to 1.2.1: https://github.com/kubernetes-csi/external-snapshotter/blob/release-1.2/CHANGELOG-1.2.md#notable-changes-1

Comment (msau42) [reply CHANGELOG-1.3.md]: We should still include it here since it's not in 1.2.0?

Comment (xing-yang) [reply CHANGELOG-1.3.md]: ok.  I don't know that we should include things that are already backported.  let me check if there are other things missing.

Comment (msau42) [reply CHANGELOG-1.3.md]: Yeah since this is the first new minor release, we include everything since 1.2.0

Comment (xing-yang) [reply CHANGELOG-1.3.md]: Changed start-sha to the commit for 1.2.0 and re-run release-notes.  Now I should have everything since then.

**Comment by xing-yang** [comment]
@msau42 Changed the start-sha to the 1.2.0 commit, instead of the 1.2.2 commit I used earlier.  Now it has picked up all commits since 1.2.0.  PTAL.

File: CHANGELOG-2.0.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+### Bug Fixes
+
+- Added extra verification of source PersistentVolumeClaim before creating snapshot. ([#172](https://github.com/kubernetes-csi/external-snapshotter/pull/172), [@xing-yang](https://github.com/xing-yang))
+- Fixes issue when SelfLink removal is turned on in Kubernetes. ([#160](https://github.com/kubernetes-csi/external-snapshotter/pull/160), [@msau42](https://github.com/msau42))
+- Snapshotter will no longer call ListSnapshots if the CSI Driver does not support this operation. ([#138](https://github.com/kubernetes-csi/external-snapshotter/pull/138), [@ggriffiths](https://github.com/ggriffiths))
+
+### Other Notable Changes
+
+- Removes createSnapshotContentRetryCount and createSnapshotContentInterval
```
Comment (msau42) [inline CHANGELOG-2.0.md]: This is also a breaking change

Comment (xing-yang) [reply CHANGELOG-2.0.md]: Moved to Breaking Changes section.

**Pull request merged**